code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -594,3 +594,12 @@
 	icon = 'icons/obj/clothing/clockwork_garb.dmi'
 	icon_state = "clockwork_cuirass_old"
 	armor = list("melee" = 5, "bullet" = 0, "laser" = -5, "energy" = 0, "bomb" = 10, "bio" = 0, "rad" = 0, "fire" = 20, "acid" = 20)
+
+/obj/item/clothing/suit/ghost_sheet
+	name = "ghost sheet"
+	desc = "The hands float by themselves, so it's extra spooky."
+	icon_state = "ghost_sheet"
+	item_state = "ghost_sheet_item"
+	w_class = WEIGHT_CLASS_TINY
+	flags_inv = HIDEGLOVES|HIDEMASK|HIDEEARS|HIDEFACE|HIDEHAIR|HIDEFACIALHAIR
+	alternate_worn_layer = 6	//I think this should be right? Same layer as head items, so hats will display over it I think.

```

Comment:
I'm not sure if same layer will render properly but you need to use a define here regardless. You'll probably want to define a UNDER_HEAD_LAYER

Reply:
Ok done, but the (X_LAYER + 1) thing confuses me, would this be identical to 7 / MASK_LAYER?
Although thinking about it, it makes more sense to do it with the defines because then it's futureproof if the layer numbers change.